dll/win32/shell32/CDefView.cpp

```diff
@@ -2151,6 +2152,24 @@ HRESULT CDefView::InvokeContextMenuCommand(CComPtr<IContextMenu>& pCM, LPCSTR lp
     cmi.lpVerb = lpVerb;
     cmi.nShow = SW_SHOW;
 
+    WCHAR szverbW[sizeof("properties")];
+    static const WORD verbmap[] = { FCIDM_SHVIEW_DELETE, FCIDM_SHVIEW_RENAME,
+                                    FCIDM_SHVIEW_PROPERTIES, FCIDM_SHVIEW_CREATELINK,
+                                    FCIDM_SHVIEW_CUT, FCIDM_SHVIEW_COPY, FCIDM_SHVIEW_INSERT };
+    for (SIZE_T i = 0; TryMapVerb && i < _countof(verbmap); ++i)
+    {
+        if (cmi.lpVerb != MAKEINTRESOURCEA(verbmap[i]))
+            continue;
+        if (PCSTR pszverbA = MapFcidmCmdToVerb((SIZE_T)cmi.lpVerb))
```

Comment:
This would avoid the cast while allowing the cmdId parameter of the function to stay a 32-bit number.
```suggestion
        if (PCSTR pszverbA = MapFcidmCmdToVerb(verbmap[i]))
```